.mergify.yml

```diff
@@ -1,46 +1,34 @@
 pull_request_rules:
-  - name: 'rebase unreviewed non-release PRs'
+  - name: "rebase unreviewed non-release PRs"
```

Comment:
Be careful when it comes to strings in YAML, especially when you say we could remove them entirely:

https://twitter.com/rogpeppe/status/1235958742828699649

😉

When it comes to single vs. double quotes: Probably makes sense to pick one variant and then escape accordingly.

Reply:
Ok, sure, keeping them does not hurt. But since we use double quotes below, I would stick to them `"`&mdash;besides, I would need to reconfigure my formatter in the other case.